modules/prebidServerBidAdapter.js

```diff
@@ -734,7 +735,13 @@ export function PrebidServer() {
       utils.logError('error parsing response: ', result.status);
     }
 
-    done();
+    const videoBid = bids.some(bidResponse => bidResponse.bid.mediaType === 'video');
+    const cacheEnabled = config.getConfig('cache.url');
+
+    // video bids with cache enabled need to be cached first before they are considered done
```

Comment:
is this a separate change?

Reply:
No those changes are part of this PR.  They are needed to ensure pbs video bids that are using prebid cache are properly used in the auction.  Without this change; the adapter calls the `done()` before the caching process has finished which ends the auction prematurely.